cernopendata/modules/fixtures/data/records/cms-primary-datasets-HIpp-Run2011A.json

```diff
@@ -40,6 +40,20 @@
       "size": 6570934150548
     },
     "experiment": "CMS",
+    "files": [
+      {
+        "checksum": "adler32:c82b6958",
+        "size": 410028,
+        "type": "index.json",
+        "uri": "root://eospublic.cern.ch//eos/opendata/cms/Run2011A/AllPhysics2760/RECO/16Jul2011-v1/file-indexes/CMS_Run2011A_AllPhysics2760_RECO_16Jul2011-v1_root_file_index.json"
```

Comment:
@katilp Since these files are RECO, I have created only one index file for all the volumes and subvolumes, as we did in the past for the other HI data.  Is this OK?

Reply:
Yes, that's fine, thanks!